### PR TITLE
fixed pr

### PR DESCRIPTION
**Pull request by OhTanishJain**

### CODE REVIEW

**Comment by so-frontend-integrations** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **ashishsatiso**
You can assign the PR to them by writing `/assign @ashishsatiso` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=scheduleonce%2Fng-strictify).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/scheduleonce/ng-strictify/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by sohimanshu — APPROVED** [review]

**Comment by so-frontend-integrations** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[sohimanshu](https://github.com/scheduleonce/ng-strictify/pull/21#pullrequestreview-980075305 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=scheduleonce%2Fng-strictify).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/scheduleonce/ng-strictify/blob/master/OWNERS)~~ [sohimanshu]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by so-frontend-integrations** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[sohimanshu](https://github.com/scheduleonce/ng-strictify/pull/21#pullrequestreview-980075305 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=scheduleonce%2Fng-strictify).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/scheduleonce/ng-strictify/blob/master/OWNERS)~~ [sohimanshu]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
